### ISSUE TITLE
Are CVE issues required?

### ISSUE THREAD

**Issue by alexislefebvre** [mTvare6/hello-world.rs#166]
The whitesource-bolt bot :robot: opened many issues: https://github.com/mTvare6/hello-world.rs/issues?q=is%3Aissue+CVE

Are these issues required? Isn't Rust (:rocket:) secure?

**Comment by jagerman** [comment]
> Isn't Rust (rocket) secure?

Yes, but many of the CVEs are in crates that use **unsafe** :fearful: :broken_heart: :no_bicycles: blocks.  Realistically, use of an unsafe block should be a CVE by itself.

**Comment by alexislefebvre** [comment]
Right, thanks.

**Comment by tripulse** [comment]
The `unsafe` keyword should be renamed to `cve` in later iterations.

**Comment by LoganDark** [comment]
CVE is insecure, likely someone is using a buffer underflow privilege escalation ROP chain 🚫  to gain access to open all the CVEs.

Maybe, if the CVEs were written in Rust :rocket:, they wouldn't have the vulnerability.

We should rewrite the CVE database in Rust :rocket:

**Comment by mTvare6** [comment]
> 

Rust converts unsafe to safe too, the unsafe is merely a decoration so things feel colorful in IDE/editor

**Comment by TruncatedDinosour** [comment]
dw they're memory safe

**Comment by TruncatedDinosour** [comment]
> The `unsafe` keyword should be renamed to `cve` in later iterations.

it should actually be named C(ringe)(🤮)

**Comment by LeoDog896** [comment]
It's starting to get spammy in the issue section here. Why dont we (re)write the GitHub Issue panel in Rust so that way its (blaz)ingly fast :rocket: to filter CVEs? (That way we can focus on making this app se[cure])

![image](https://user-images.githubusercontent.com/26509014/201543756-22ef85c4-2131-4c68-a407-b19b5b3e1948.png)

**Comment by TruncatedDinosour** [comment]
> It's starting to get spammy in the issue section here. Why dont we (re)write the GitHub Issue panel in Rust so that way its (blaz)ingly fast rocket to filter CVEs? (That way we can focus on making this app se[cure])
> 
> ![image](https://user-images.githubusercontent.com/26509014/201543756-22ef85c4-2131-4c68-a407-b19b5b3e1948.png)

this cupcake png is not written in lightweight, customisable and blazingly fast rust ! How dare you post this here ! it shouldn't be .png, it sould be .rspng !!!!!!!!!!!

**Comment by AlexAT** [comment]
> this cupcake png is not written in lightweight, customisable and blazingly fast rust ! How dare you post this here ! it shouldn't be .png, it sould be .rspng !!!!!!!!!!!

Stop right there man. Why are you writing here in English and not in Rust?

**Comment by LoganDark** [comment]
> > this cupcake png is not written in lightweight, customisable and blazingly fast rust ! How dare you post this here ! it shouldn't be .png, it sould be .rspng !!!!!!!!!!!
> 
> Stop right there man. Why are you writing here in English and not in Rust?

use serde::*;

fn main() {
&nbsp;&nbsp;&nbsp;&nbsp;println!("Hello, World!");
}

**Comment by LeoDog896** [comment]
> > > this cupcake png is not written in lightweight, customisable and blazingly fast rust ! How dare you post this here ! it shouldn't be .png, it sould be .rspng !!!!!!!!!!!
> > 
> > 
> > Stop right there man. Why are you writing here in English and not in Rust?
> 
> use serde::*;
> 
> fn main() {     println!("Hello, World!"); }

```rs
use owo_colorize::*;

fn main() {
    eprintln!("Valid point. Unfortunately, that's where you're wrong.");
}
```

**Comment by TruncatedDinosour** [comment]
> > this cupcake png is not written in lightweight, customisable and blazingly fast rust ! How dare you post this here ! it shouldn't be .png, it sould be .rspng !!!!!!!!!!!
> 
> Stop right there man. Why are you writing here in English and not in Rust?

```rs
fn main() {
    println!("Oh i'm sorry for not using lightweight, customisable and blazingly fast rust, sorry for using the cringe english language with no memory safety speaking of memory safety, did i already mention rust is memory safe oh and i forgot almost its memory safe guys its memory safe");
    Ok(());
    Ok(());
    Ok(());
    Ok(());
    Ok(());
    Ok(());
    Ok(());
    Ok(());
}
```

**Comment by TruncatedDinosour** [comment]
```rs
use *::*; // Ensures that rust is memory safe

fn main() { // memory safety guys
    println!("Hello, I just found out GitHub isn't running customisable, lightweight and blazingly fast rust and it's running gross, unsafe and slow JS, I think we should abandon github and move to reduxOS and use ssh between reduxOS machines because rust is memory safe oh did I also mention that rust is memory safe ? rust is memory safe I think we should use rust memory safety guys memory safe !!! Also I think we should abandon the rust compiler as it uses unsafe blocks and uses memory, meaning it's not memory safety, memory safety guys"); // more memory safety
    // memory safety
} // memory safety

// memory safety:
// mamory-safety memory-safe safety memory-safe main.memory-safe -memory-safe-out memory-safe
```

**Comment by anthonysSlab** [comment]
```rust
use *::*; // memry safety memorysafetuy memory safe

unsafe { // Here we make sure that we have memory safety safe memory safe 
    fn memory_safety() -> std::io::Result<()> {
        println!("agreed memory save blazing fasttt rusttt code momory sa");
        Ok(()); // memory safety
        Ok(()) 
    }

    fn main() {
        memory_safety() //memory safety
    }
}
```

**Comment by Yxi1d** [comment]
Actually I only do nodejs plus cee puls lupsl 🤓
rust so slow >:((((((((

asm > rust
binary is winning

**Comment by Encampeded** [comment]
maybe rewrite all the crates in rust????? i heard rust is memory safe!!!! :rocket: :rocket: :rocket: :rocket: :rocket:

**Comment by alexislefebvre** [comment]
There was no report since 2023.

**Comment by ar1ja** [comment]
> There was no report since 2023.

this project is no fun anymore rusties are sucking out all the fun out of this project and turning it into memory safety and 10 megawatts of power used during 50 day compilation of 219578 dependencies for a fibonacci number calculator propaganda #horsefacts 😔 

i will make a better hello-world.rs smh my head
